test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryStoreTest.cs

```diff
@@ -23,5 +23,31 @@ protected override void AddRoleStore(IServiceCollection services, object context
         {
             services.AddSingleton<IRoleStore<IdentityRole>, InMemoryRoleStore<IdentityRole>>();
         }
+
+        protected override InMemoryUser CreateTestUser(string username = "", string email = "",
```

Comment:
Isn't InMemoryUser the same as the new shared test user now?  Can we switch this over to use the common POCO?